modules/apps/account/account-test/src/testFunctional/tests/2BAccount.testcase

```diff
@@ -722,6 +722,46 @@ definition {
 		}
 	}
 
+	@description = "This test asserts an account member can search accounts they belong to"
+	@priority = "3"
+	test AccountMemberCanSearchForAnAccount {
+		task ("Given a user is added to an additional account") {
```

Comment:
- Could you please double check if the steps here can be replaced by the exist json macro? Like add account via json, and add account user via json. 
- If so, we probably also no need to login the admin user. 
- Since we are testing for the _account member_ permission, we can add a new user, and assign him at least 2 accounts for testing.
Thanks.

Reply:
I think there will be an issue assigning the user to different accounts through JSON. Right now, we have _JSONAccountEntryUser.addAccountEntryUser_ which creates a user AND adds that user into an account.
As far as I know, this macro can only be used to add multiple different users into any account, but not one user into multiple different accounts. 

I think I can try:
1. (Staying logged in as userea): Create a new account user for Account Name through JSON.
2. Create another account through JSON.
3. Assign new user to 2nd account through UI (2B page).

Reply:
I have tried testing many times here: https://github.com/epark18/liferay-portal/pull/29 using the JSON macros while staying logged in as userea with no luck 😢 Main issue include assigning the new user to a 2nd account. Pushed up a working test, but I had to sign in as Test.